dom-manipulation/main.js

```diff
@@ -0,0 +1,25 @@
+let totalClicks = 0;
+const $hotButton = document.querySelector('.hot-button');
+console.log($hotButton);
```

Comment:
Delete all `console.log` before submitting! They are only for testing / debugging purposes. Unless we specifically ask for them!